docs/API/metrics_api.md

```diff
@@ -0,0 +1,155 @@
+## <a name="top"></a>Metrics API
```

Comment:
Anchor tags like this can really screw up ReadtheDocs - please remove
(If really necessary place outside of the header)

Reply:
Ok. Fixed in ef4582bceac76559b84a45f17bef934c90ce6529